site/docs/v1/tech/apis/scim/_scim-enterpriseuser-response-body.adoc

```diff
@@ -0,0 +1,69 @@
+Because SCIM schemas are completely customizable, The SCIM response returned is generated from the result of SCIM User object populated by your outgoing response lambda.
+
+The following is an example of a typical SCIM User response body that your SCIM client might expect. However, your outgoing response lambda could generate a completely different SCIM schema is desired.
+
+This is taken from the SCIM Schema RFC describing a SCIM User schema. For the full specification you can find the RFC here:
+
+- https://datatracker.ietf.org/doc/html/rfc7643#page-19
+
+==== Response Body
+[.api]
```

Comment:
You kind of make this serve two purposes:

* suggested mappings from the FusionAuth user object
* a sample response body, from the spec perhaps?

I'd suggest doing just the second, and then building a table that offers a typical mapping. This could be a different section of the API docs.

Reply:
Not sure I understand your suggestion exactly. You want me to just remove all the [field] descriptions and just have a plain JSON response body with nothing else?

Reply:
Well, if we remove the advice on mapping then it's really just a copy of the spec. Maybe we should jsut link to the spec and not provide an example request and response JSON?